assets/bingo/lang/en_us.json

```diff
@@ -201,6 +201,9 @@
 	"bingo.lobby.settings.pregen_behavior.turn_on": "Switch to “On game start”",
 	"bingo.lobby.settings.pregen_behavior.turn_off": "Switch to “On card generation”",
 
+	"bingo.lobby.menu.back": "Back",
+	"bingo.lobby.menu.back.hover": "Go back to previous page",
+	
```

Comment:
```suggestion
```
What would these be for? They are not used anywhere.

Reply:
Artefacts from a temporary translation